packages/autocomplete-js/src/__tests__/reverseHighlightHit.test.ts

```diff
@@ -0,0 +1,109 @@
+import { reverseHighlightHit } from '../highlight';
+
+describe('reverseHighlightHit', () => {
+  test('returns a reversed partially highlighted Recent Search hit', () => {
```

Comment:
Those tests are not specific to recent searches.

```suggestion
  test('returns a reversed partially highlighted hit', () => {
```

Reply:
Fixed in https://github.com/algolia/autocomplete.js/pull/388/commits/e59e4d4b483077d35aa6b6df44094d35098b9d7a